src/core/Table/TablePaginator.tsx

```diff
@@ -89,6 +92,11 @@ export type TablePaginatorProps = {
      * @default (page: number) => `Go to page ${page}`
      */
     goToPageLabel?: (page: number) => string;
+    /**
+     * A label shown next to the page size selector. Use `null` to hide.
```

Comment:
Maybe add test that label is hidden when passing in null or empty string

Reply:
added test 🤷‍♀️